### ISSUE TITLE
Error on radar_setup

### ISSUE THREAD

**Issue by N2WU** [openradarinitiative/openradar_mmwave_utils#2]
Having an issue when I run setup_radar. I get successful connection, but have the error
`mmWave Device Power on failed for deviceMap 1 with error -8`

Any tips? I have the board connected correctly in the SOP0 position. However, I only have one microusb connected to the DCA1000 (to j4) and one connected to the AWR, when there are two microusb ports on the DCA1000 (j1 is the other one).

**Comment by jpanda001** [comment]
@danielgusland @N2WU Hello, I am also facing this issue as well. As mentioned in the manual, I have connected usb to Radar_FTDI port and have set the jumper only to SOP0 position. Any help will be greatly appreciated

**Comment by danielgusland** [comment]
Have you tried running it with sudo?

**Comment by jpanda001** [comment]
Yes, as you mentioned I used `sudo` to run the `./setup_radar` executable. I followed your instructions line to line including installing the FTDI drivers and all listed dependencies. I still keep on getting `mmWave Device Power on failed for deviceMap 1 with error -8`. 

Following is the complete output I get:
```
================= mmWaveLink Example Application ====================

====================== SPI Mode of Operation ======================

AR-DevPack-EVM-012 A
AR-DevPack-EVM-012 B
AR-DevPack-EVM-012 C
AR-DevPack-EVM-012 D
Device map 1 : SOP 4 mode successful

Device map 1 : Device reset successful

rlDeviceEnable Callback is called by mmWaveLink for Device Index [0]

mmWave Device Power on failed for deviceMap 1 with error -8

=========== mmWaveLink Example Application execution Failed ===========
```
Also, which linux distro and what version did you use? I'm using Ubuntu 20.04

**Comment by N2WU** [comment]
Also, I am unable to compile the DCA1000 directory. I get the error

 `CMake Error at CMakeLists.txt:9 (add_executable):`
`Cannot find source file:`
`Common/Json_Utils/dist/json/json.h`
`Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp .hxx .in .txx`

`CMake Error: CMake can not determine linker language for target: setup_dca_1000`
`CMake Error: Cannot determine link language for target "setup_dca_1000". `

**Comment by danielgusland** [comment]
I am sorry for not responding, I have been very busy. 
I will try to help you guys out in the coming days. 

It might be that you have the "new" version of the the AWR2243 boost. 
Can you please try to setup the radar using the code in the branch `AWR2243_ES1_1` instead of the `master` branch?

**Comment by danielgusland** [comment]
> 

This error is now fixed. 

Somehow the json utils were left out, they are added in commit f9f1665.

**Comment by jpanda001** [comment]
Hello @danielgusland, thanks for responding and sorry for the late response from my end. I used the ARW2243_ES1_1 branch and I can connect to the radar now. setup_radar runs successfully. I'm just having some problems running the DCA1000EVM setup but its probably because of wrong network setup on my end. If I'm still stuck and make no progress, I'll get back to you.

Thanks for all the help and creating this driver in the first place. You are saving lives of undergrad students

**Comment by danielgusland** [comment]
> Hello @danielgusland, thanks for responding and sorry for the late response from my end. I used the ARW2243_ES1_1 branch and I can connect to the radar now. setup_radar runs successfully. I'm just having some problems running the DCA1000EVM setup but its probably because of wrong network setup on my end. If I'm still stuck and make no progress, I'll get back to you.
> 
> Thanks for all the help and creating this driver in the first place. You are saving lives of undergrad students

That is good!

The network setup should be pretty straight forward. 
Make sure that the fixed ip is set when you power up the DCA1000EVM.
